website/docs/docs/core/about-the-cli.md

```diff
@@ -4,14 +4,12 @@ id: "about-the-cli"
 sidebar_label: "About the CLI"
 ---
 
-dbt ships with a command line interface (CLI) for running your dbt project. This way of running dbt a dbt project is free and open source.
+dbt ships with a command line interface (CLI) for running your dbt project. This way of running dbt and a dbt project is free and open source.
 
 To use the CLI, your workflow generally looks like:
-* **Build your dbt project in a code editor:** popular choices include VSCode and Atom.
-* **Run your project from the command line:**
-  * macOS ships with a default Terminal program, however you can also use iTerm or the command line prompt within a code editor to execute dbt commands
+1. **Build your dbt project in a code editor &mdash;** popular choices include VSCode and Atom.
 
-The CLI is also available for dbt Cloud.  Additional components must be installed for the CLI to communicate via dbt Cloud APIs.  For more information, visit the [dbt Cloud CLI GitHub repository](https://github.com/data-mie/dbt-cloud-cli)
```

Comment:
removed this as it's not dbt supported. convo [here](https://dbt-labs.slack.com/archives/C02G5V9JBHV/p1682434162754379?thread_ts=1682429905.097399&cid=C02G5V9JBHV)